test/jdk/java/lang/String/CompactString/EqualsIgnoreCase.java

```diff
@@ -75,4 +77,32 @@ public void testEqualsIgnoreCase(String str, String anotherString,
                                             source));
                         });
     }
+
+    /**
+     * Exhaustively check that all 256x256 latin1 code point pairs are equalsIgnoreCased
+     * in a manner consistent with Character.toLowerCase(Character.toUpperCase(c));
+     */
+    @Test
+    public void checkConsistencyWithCharacterUppercaseLowerCase() {
+        for (int ab = 0; ab < 256; ab++) {
+            for (int bb = 0; bb < 256; bb++) {
+                char a = (char) ab, b = (char) bb;
```

Comment:
char is an unsigned numeric type, so cleaner is

for (char a = 0; a < 256; a++) 
for (char b = 0; b < 256; b++)

Reply:
Thanks, fixed. Might have been copied over from processing of code points in the higher planes. Not needed here.